src/allmydata/introducer/client.py

```diff
@@ -115,18 +115,26 @@ def connect_failed(failure):
         d.addErrback(connect_failed)
 
     def load_announcements(self, storage_broker):
-        if os.path.exists(self.config_file):
-            f = open(self.config_file)
+        if self.config_filepath.exists():
+            with self.config_filepath.open() as f:
```

Comment:
The ``f.close()`` isn't needed (that's the point if the context manager), and ``server_params = ...`` should be indented.